src/main/resources/spell_teleports.tsv

```diff
@@ -0,0 +1,158 @@
+
+#
+# Lunar Spellbook - Varbit 4070=2
+#
+
+# Moonclan Teleport
+2113 3915 0	557,563,9075;563,9075,4696;563,9075,4698;563,9075,4699;563,9075,1385;563,9075,20736;563,9075,6562;563,9075,3053	69 Magic	Lunar Diplomacy	4	Moonclan Teleport	20	4070=2
+
+# Ourania Teleport
+2468 3246 0	557,563,9075;563,9075,4696;563,9075,4698;563,9075,4699;563,9075,1385;563,9075,20736;563,9075,6562;563,9075,3053	71 Magic	Lunar Diplomacy	4	Ourania Teleport	20	4070=2
+
+# Waterbirth Teleport
+2546 3756 0	555,563,9075;563,9075,4698;563,9075,4694;563,9075,4695;563,9075,1383;563,9075,20730;563,9075,6562;563,9075,11787;563,9075,21006;563,9075,25574	72 Magic	Lunar Diplomacy	4	Waterbirth Teleport	20	4070=2
+
+# Barbarian Teleport
+2543 3569 0	554,563,9075;563,9075,4699;563,9075,4697;563,9075,4694;563,9075,1387;563,9075,11787;563,9075,3053;563,9075,11998;563,9075,20714	75 Magic	Lunar Diplomacy	4	Barbarian Teleport	20	4070=2
+
+# Khazard Teleport
+2636 3167 0	555,563,9075;563,9075,4698;563,9075,4694;563,9075,4695;563,9075,1383;563,9075,20730;563,9075,6562;563,9075,11787;563,9075,21006;563,9075,25574	78 Magic	Lunar Diplomacy	4	Khazard Teleport	20	4070=2
+
+# Fishing Guild Teleport
+2613 3391 0	555,563,9075;563,9075,4698;563,9075,4694;563,9075,4695;563,9075,1383;563,9075,20730;563,9075,6562;563,9075,11787;563,9075,21006;563,9075,25574	85 Magic	Lunar Diplomacy	4	Fishing Guild Teleport	20	4070=2
+
+# Catherby Teleport
+2801 3449 0	555,563,9075;563,9075,4698;563,9075,4694;563,9075,4695;563,9075,1383;563,9075,20730;563,9075,6562;563,9075,11787;563,9075,21006;563,9075,25574	87 Magic	Lunar Diplomacy	4	Catherby Teleport	20	4070=2
+
+# Ice Plateau Teleport
+2975 3938 0	555,563,9075;563,9075,4698;563,9075,4694;563,9075,4695;563,9075,1383;563,9075,20730;563,9075,6562;563,9075,11787;563,9075,21006;563,9075,25574	89 Magic	Lunar Diplomacy	4	Ice Plateau Teleport	20	4070=2
+
+
+#
+# Arceuus Spellbook - Varbit 4070=3
+#
+
+# Arceuus Library Teleport
+1632 3834 0	563,557;563,4696;563,4698;563,4699;563,1385;563,20736;563,6562;563,3053	6 Magic		4	Arceuus Library Teleport	20	4070=3
+
+# Draynor Manor Teleport
+3108 3351 0	563,557,555;563,555,4696;563,4698;563,555,4699;563,557,4694;563,557,4695;563,557,1383;563,557,20730;563,557,11787;563,557,21006;563,555,1385;563,555,20736;563,555,3053;563,6562;563,557,25574;563,1385,25574;563,20736,25574;563,3053,25574	17 Magic		4	Draynor Manor Teleport	20	4070=3
+
+# Battlefront Teleport
+1347 3741 0	563,557,554;563,554,4696;563,554,4698;563,4699;563,557,4697;563,557,4694;563,554,1385;563,554,20736;563,554,6562;563,557,1387;563,557,11787;563,557,11998;563,3053;563,557,20714;563,1385,20714;563,20736,20714;563,6562,20714	23 Magic		4	Battlefront Teleport	20	4070=3
+
+# Mind Altar Teleport
+2976 3509 0	563,558	28 Magic		4	Mind Altar Teleport	20	4070=3
+
+# Respawn Teleport
+# TODO: Figure out values of Varbit 668
```

Comment:
I figured out varbit 668 is used to determine which respawn point to use but i couldnt figure out which values meant which place. I assume the order is the same from the [wiki page](https://oldschool.runescape.wiki/w/Spawning) but i couldnt verify.